tests/python/rest_api/test_tasks.py

```diff
@@ -5631,21 +5631,23 @@ def test_can_import_datumaro_json(self, admin_user, tasks, dimension):
     @pytest.mark.parametrize(
         "format_name",
         [
-            "COCO 1.0",
-            "COCO Keypoints 1.0",
+            # FIXME: RejectionReason.detection_unsupported
+            # "COCO 1.0",
+            # "COCO Keypoints 1.0",
             "CVAT 1.1",
             "LabelMe 3.0",
             "MOT 1.1",
-            "MOTS PNG 1.0",
-            "PASCAL VOC 1.1",
+            "MOTS PNG 1.0",  # RejectionReason.unmet_requirements, but specific requirement information unavailable
+            # FIXME: looks like it desn't fail
+            # "PASCAL VOC 1.1",
```

Comment:
```suggestion
            pytest.param("PASCAL VOC 1.1", marks=pytest.mark.xfail),
```

This will make sure that if the behavior is fixed, the test is updated.